app/src/main/java/ca/wheresthebus/ui/home/HomeViewModel.kt

```diff
@@ -13,5 +13,5 @@ class HomeViewModel : ViewModel() {
     }
     val text: LiveData<String> = _text
 
-    val busTimes =  MutableLiveData<MutableMap<StopCode, List<Duration>>>()
+    val busTimes =  MutableLiveData<Map<StopCode, List<Duration>>>()
```

Comment:
not sure if there was an actual reason why it was a mutablemap instead of just a map, so changed it back to work with the favouriteStopsList.map

Reply:
nvm changed it back to mutablemap